src/components/Shared/PreTransactionModal.scss

```diff
@@ -154,16 +154,101 @@
   font-size: 11px;
   position: relative;
   display: inline-block;
-  left: -10px;
+  text-align: left;
+  line-height: 1.2em;
+}
+
+div.exchangeList {
+  color: rgba(5, 113, 255, 1.000);
+  font-size: 11px;
+  width: auto;
+  position: relative;
+  text-align: left;
+  transform: translate(0,0);
+  display: inline-block;
+  margin: 0;
+  padding-right: 25px;
+
+  ul {
+    position: absolute;
+    top: -130px;
+    left: 0;
+    background-color: white;
+    list-style: none;
+    box-shadow: $shadow-3;
+    margin: 0;
+    width: 120px;
+    left: -1px;
+    padding: 10px 0;
+    opacity: 0;
+    transform: scale(0);
+    transition: all .25s ease;
+    background-color: rgba(255,255,255,1);
+    border-radius: 0 10px 10px 0;
+
+    .diamond {
+      width: 10px;
+      height: 10px;
+      background-color: #fff;
+      position: absolute;
+      z-index: 0;
+      top: 50%;
+      transform: translateY(-5px) rotate(45deg);
+      left: -5px;
+    }
+
+    li {
+      position: relative;
+      z-index: 100;
+    }
+
+    a {
+      font-weight: bold;
+      color: rgba(80, 97, 118, 1.000);
+      display: block;
+      padding: 5px 10px;
+
+      b {
```

Comment:
b?

Reply:
? b = bold. i just moved these styles. didnt change them

Reply:
yeah, I knew bold, just surprised that anyone is still using it.